utils/create-keying-sheet-tfm-facilities/mock-data/deal.ts

```diff
@@ -0,0 +1,262 @@
+import { Deal } from '@ukef/dtfs2-common';
+import { ObjectId } from 'mongodb';
+
+/**
+ * Creates an active deal that can be used for utilisation reporting testing.
+ * It can be replaced by creating a deal manually and then pulling from the mongo db collection.
+ * This way we ensure we are inserting a valid deal.
+ * @param ukefDealId - ukef deal id to use
+ * @param dealId - id of deal
+ * @param portalUserId - id of a portal user
+ * @returns a deal
+ */
+export const aDeal = (ukefDealId: string, dealId: ObjectId, portalUserId: ObjectId): Deal => ({
```

Comment:
Love this function, think it's definitely the best way to do this seeding even if it might go out of date. Just wanted to comment to say writing the shape of deals into this format so you can see what it is clearly makes me realise it's grim aha